### ISSUE TITLE
Search indexes not getting saved when adding a new site to a section

### ISSUE THREAD

**Issue by brandonkelly** [craftcms/cms#6296]
Hi, I create a new site, Craft auto copy "Entries" are finished, But these entries not have search indexs, We use code `craft.entries({section: 'caseStudies',type: 'caseStudies',limit: 12,search: {query: platforms:office360}});`, But the length of result is zero,  I try change the value of field in entries, And save, Then the result have content. So I think craft do not build the search indexs for the entries of new site. Is it?

_Originally posted by @DandeSuen in https://github.com/craftcms/cms/issues/5859#issuecomment-652320862_

**Comment by brandonkelly** [comment]
@DandeSuen We just released Craft 3.4.26 with a fix for this. You can manually update the search indexes for your existing entries by running the following CLI command:

```sh
./craft resave/entries --section=mySectionHandle --update-search-index
```

**Comment by DandeSuen** [comment]
Hi @brandonkelly, That's great! Thanks a lot.
